File: 백준/Bronze/4796. 캠핑/캠핑.java
import java.io.*;
import java.util.*;

public class Main{
    public static void main(String[] args) throws IOException{

        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer st;
        StringBuilder sb = new StringBuilder();

        int n = 0;
        int answer;
        int last;
        while(true){
            n++;
            st = new StringTokenizer(br.readLine());
            int L = Integer.parseInt(st.nextToken());
            int P = Integer.parseInt(st.nextToken());
            int V = Integer.parseInt(st.nextToken());

            if(L==0 && P==0 && V==0){
                break;
            }

            answer = L*(V/P);
            last = V%P;
            if (last>=L){
                answer+=L;
                sb.append("Case "+n+": "+answer).append("\n");
            }
            else {
                answer+=last;
                sb.append("Case "+n+": "+answer).append("\n");
            }

        }
        System.out.println(sb.toString());
    }
}

File: 백준/Bronze/4796. 캠핑/캠핑.py
n = 0
while True:
    answer = 0
    n += 1
    L, P, V = map(int, input().split())
    if L == 0 and P == 0 and V == 0:
        break
    answer += L * (V // P)
    if V % P >= L:
        answer += L
    else:
        answer += V % P
    print(f"Case {n}: {answer}")